packages/desktop/components/AccountManagementDetails.svelte

```diff
@@ -52,21 +54,19 @@
     $: accountId = isAccountOutput(selectedOutput) ? (selectedOutput.output as AccountOutput)?.accountId : null
     $: address = accountId ? AddressConverter.addressToBech32(new AccountAddress(accountId)) : null
     $: isMainAccount = accountId && accountId === $selectedWalletMainAccountId
-    $: balance = getAccountBalance(selectedOutput, isImplicitAccount)
-    $: formattedBalance = balance ? formatTokenAmountBestMatch(balance, getBaseToken()) : '-'
+    $: implicitAccountBalance = getImplicitAccountBalance(selectedOutput)
+    $: formattedImplicitAccountBalance = implicitAccountBalance
+        ? formatTokenAmountBestMatch(implicitAccountBalance, getBaseToken())
+        : '-'
     $: hasStakingFeature = hasOutputStakingFeature(selectedOutput)
     $: rawStakedAmount = getStakedAmount(selectedOutput)
     $: formattedStakedAmount = formatTokenAmountBestMatch(rawStakedAmount, getBaseToken())
     $: primaryKey = $selectedWallet?.primaryKey
     $: listBlockKeysFeature(selectedOutput)
+    $: hasMainAccountNegativeBIC = $selectedWallet?.balances?.blockIssuanceCredits?.[$selectedWallet?.mainAccountId] < 0
 
-    function getAccountBalance(outputData: OutputData, isImplicitAccount: boolean): number | undefined {
-        if (isImplicitAccount) {
-            return Number(outputData.output.amount)
-        } else {
-            // TODO: Calculate the balance of an account output https://github.com/iotaledger/firefly/issues/8080
-            return undefined
-        }
+    function getImplicitAccountBalance(outputData: OutputData): number | undefined {
```

Comment:
because this is implicit account only, you should not return a balance if the output is not an implicit account